products/mattermost.md

```diff
@@ -14,12 +14,16 @@ auto:
 releases:
 -   releaseCycle: "7.7"
     eol: 2023-04-15
-    releaseDate: 2023-01-16
```

Comment:
The Mattermost description could explain the differences between Mattermost Enterprise and Team.

It could also mention the [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html).

Reply:
#2319